XRTK-Core/Packages/com.xrtk.core/Services/MixedRealityToolkit.cs

```diff
@@ -574,7 +574,8 @@ private static void EnsureMixedRealityRequirements()
 
                 availablePlatforms.Add(platformInstance);
 
-                if (platformInstance.IsAvailable)
+                if (platformInstance.IsActive ||
```

Comment:
I am thinking whether this shouldn't be `&&` instead of `||`? With `||` if my service is configured for Editor, Android it will run as soon as I am in editor, right? But with `&&` we get "it only runs when in editor and the build target is Android". Do we want that? I am unsure.

Reply:
If it's `&&` then the platform would never run in a build.

Reply:
:D true